tsdb/challenges/default.json

```diff
@@ -152,6 +152,56 @@
           "iterations": 100
         }
         {% endif %}
+        {% if p_index_mode == "time_series" and run_esql_aggs is defined %}
```

Comment:
Maybe add `run_esql_aggs` to list of track params in README?

Reply:
We could maybe add an entry to the it tests too - at the moment this combination wont be tested. We probably have other tracks that have areas that miss tests due to such patterns